### PR TITLE
Update Readme.md to include prerequisites.

### PR DESCRIPTION
**Pull request by JamesDrummond**
I copied over all prerequisites from https://github.com/theia-ide/yangster-electron except for Java. Please let me know if Java is required?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,13 @@ Theia is an extensible platform to develop full-fledged multi-language Cloud & D
 - Provide support for multiple languages via the language and debug server protocols
 - Provide modern GUI with javascript UI libraries
 
+
+## Prerequisites
```
Comment (akosyakov) [inline README.md]: it is already covered here: https://github.com/theia-ide/theia/blob/master/doc/Developing.md#prerequisites

**Comment by JamesDrummond** [comment]
@akosyakov Was not apparent to me to look there. I revised the readme to refer to the quick start guide. Not sure if where I put it is the right place but the start README.md should refer to this doc somewhere.

**Comment by akosyakov** [comment]
@JamesDrummond it is referenced from `Contributing`: https://github.com/theia-ide/theia#contributing

What are these prerequisites for? Theia, extension or application development? It should go to corresponding documentation.

**Comment by JamesDrummond** [comment]
@akosyakov Not sure it's contributing as I could use Theia and not be contributing. If I was using a linux system and wanted to run Theia where would I need to go to do this? 

Seems like most things on README.md is referring to extending Theia but not how to actually use as a standalone product. The only thing I see is for running via Docker.

File: README.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ Theia is an extensible platform to develop full-fledged multi-language Cloud & D
 
 ## Getting Started
 Here you can find guides and examples for common scenarios:
+- [Quick Start Guide](https://github.com/theia-ide/theia/blob/master/doc/Developing.md#prerequisites)
```
Comment (akosyakov) [inline README.md]: I would rather replace it with `Develop a new Theia application` or `Build your own IDE` pointing to https://github.com/theia-ide/theia-website/blob/1c4d6fc597dfe65104f5e01ff1a7b3497f12d768/doc/Composing_Applications.md after https://github.com/theia-ide/theia-website/pull/11 gets merged.

**Comment by akosyakov** [comment]
`Getting Started` section was updated on the master with https://github.com/theia-ide/theia/pull/2748

**Pull request closed without merging**